internal/util/aligned_ticker_test.go

```diff
@@ -0,0 +1,92 @@
+package util
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tilinna/clock"
+
+	"github.com/atlassian/gostatsd/internal/fixtures"
+)
+
+const ms = int64(time.Millisecond)
+
+func checkTime(t *testing.T, ctx context.Context, ch <-chan time.Time, expected time.Time) {
+	select {
+	case <-ctx.Done():
+		t.Error("timed out")
```

Comment:
This could be moved to `assert.FailNow(t, `timed out`)`

Reply:
I though there was a better way, but I'm so used to `require` and `assert` that I couldn't think of it.  I'll fix it.

Reply:
Oh, `assert.FailNow`, not `t.FailNow("timed out")`, I see.  Groovy.